types/auth0-js/index.d.ts

```diff
@@ -350,7 +350,7 @@ export class Redirect {
             connection: string;
             /** allow userMetadata to be passed to signUp */
             userMetadata?: unknown | undefined;
-        },
+        } & CrossOriginLoginOptions,
```

Comment:
[Redirect.signupAndLogin](https://github.com/auth0/auth0.js/blob/1a9e2a15d8bb7eb3fe993385ea4405a94487aac3/src/web-auth/redirect.js#L55) passes `options` param down to [webauth.login](https://github.com/auth0/auth0.js/blob/1a9e2a15d8bb7eb3fe993385ea4405a94487aac3/src/web-auth/index.js#L854)